languages/de-DE/Docs/PaletteColours.multids

```diff
@@ -29,6 +29,8 @@ external-link-foreground-hover: Externer Link Vordergrund (hover)
 external-link-foreground-visited: Externer Link besucht Vordergrund
 external-link-foreground: Externer Link Vordergrund
 foreground: Vordergrund Global
+menubar-background: Oberes Menü Hintergrund
+menubar-foreground: Oberes Menü Vordergrund
```

Comment:
"Oberes" is the same as "Top" and Jeremy wanted to avoid it, since it can be set to be "Bottom" or Left or Right or whatever. So imo it should only be "Menü Hintergrund". 

We could call it "Dynamishes Menü Hitergrund"

Reply:
ah now I get it :P

Reply:
Let's keep it simple with "Menü Hintergrund" / "Menü Vordergrund"

Reply:
ok

Reply:
thanks @pmario